sections/layer2/deposit/DepositTab/TabContent.tsx

```diff
@@ -54,7 +54,7 @@ const TabContent: FC<TabContentProps> = ({
 	const currencyKey = CryptoCurrency['SNX'];
 
 	const renderButton = () => {
-		if (depositAmount) {
+		if (depositAmount && !depositAmount.isZero()) {
```

Comment:
I think `depositAmount.toNumber()` could be a more concise way of providing this condition